gemini/src/main/java/com/techempower/cache/EntityStore.java

```diff
@@ -524,7 +524,7 @@ else if (group.type().isAnnotationPresent(NotIndexed.class))
     methodValueCaches.put(group.type(), 
         new MethodValueCache<>(this, group.type()));
     
-    log.debug("Registered {} with id {}", group, group.getGroupNumber());
+    log.info("Registered {} with id {}", group, group.getGroupNumber());
```

Comment:
Changed to info level because this is useful for any app to have logged at startup.